packages/repo-doctor/src/rules/locales.js

```diff
@@ -0,0 +1,31 @@
+const fetch = require('node-fetch')
+
+const fetchRepositoryDirectoryContent = async (repoSlug, pathName) => {
+  return await fetch(
+    `https://api.github.com/repos/${repoSlug}/contents/${pathName}`
+  ).then(resp => resp.json())
+}
+
+const localesInRepo = () =>
+  async function*(repositoryInfo) {
+    const localeDirContent = await fetchRepositoryDirectoryContent(
+      repositoryInfo.slug,
+      'src/locales'
```

Comment:
I don't think this will work out, the locales for drive and photos for example are in the same repo but not the same folder, and packages such as cozy-sharing won't work either I think?

Reply:
This is why I think rules could be configured per repo.